extension/content.js

```diff
@@ -0,0 +1,91 @@
+// Load jQuery and run myCode() after loaded
```

Comment:
Did you write this loader? It would be better is jQuery was bundled into your JS asset